src/commands/hash.ts

```diff
@@ -35,20 +36,25 @@ export class HashManifest extends Command {
     await hashFiles(toHash, manifest);
 
     await fsa.write(args.manifest, Buffer.from(manifest.toJsonString()));
-    logger.info({ path: args.inputFile, count: manifest.files.size }, 'Hash:Done');
-  }
-}
+    logger.info({ path: args.manifest, count: manifest.files.size }, 'Hash:Done');
+  },
+});
 
 export async function hashFiles(toHash: ManifestFile[], manifest: ManifestLoader, Q = pLimit(5)): Promise<void> {
   let count = 0;
+  let startTime = performance.now();
```

Comment:
Is that more precise that `Date.now()`?

Reply:
Yes,

`Date.now()` gives ms percision as a int, `performance.now()` gives us precision as a floating point eg 1.324ms